src/actions/index.js

```diff
@@ -10,9 +10,16 @@ export const DECREMENT = 'DECREMENT';
 // is left to the reducer(s).
 
 export const increment = () => {
-  // Fill in this function
+  
+  return {
+    type: INCREMENT,
+    payload: 1
```

Comment:
You actually don't need a payload here. `payload` is gonna have a value of the parameter. But in this function, we actually didn't need to pass in a parameter/argument, therefore, payload is not necessary. Plus we aren't referencing this payload anywhere anyways